### PR TITLE
Use JMESPath to extract cost metadata

### PR DESCRIPTION
**Pull request by xbrianh**
depends on #238

### CODE REVIEW

File: terra_notebook_utils/workflows.py [hunk 1]
```diff
@@ -50,6 +51,12 @@ def get_workflow(submission_id: str,
     resp.raise_for_status()
     return resp.json()
 
+def _get(path: str, data: Dict[str, Any]) -> Any:
+    res = jmespath.search(path, data)
+    if res is None:
+        raise TNUCostException(f"'{path}' note found in {json.dumps(data)}")
```
Comment (aofarrel) [inline terra_notebook_utils/workflows.py:57]: If I understand this correctly, it should be:

raise TNUCostException(f"'{path}' **not** found in {json.dumps(data)}")

other than that LGTM

Comment (xbrianh) [reply terra_notebook_utils/workflows.py:57]: Thanks for catching that.

File: terra_notebook_utils/workflows.py [hunk 2]
```diff
@@ -50,6 +51,12 @@ def get_workflow(submission_id: str,
     resp.raise_for_status()
     return resp.json()
 
+def _get(path: str, data: Dict[str, Any]) -> Any:
+    res = jmespath.search(path, data)
+    if res is None:
+        raise TNUCostException(f"'{path}' not found in {json.dumps(data)}")
```
Comment (DailyDreaming) [inline terra_notebook_utils/workflows.py:57]: Consider printing with `indent=4` or `indent=2`.

**Review by DailyDreaming — APPROVED** [review]
LGTM.  See comment.

File: terra_notebook_utils/workflows.py (hunk 2 above)
Comment (xbrianh) [reply terra_notebook_utils/workflows.py:57]: will change

**Pull request merged**